esp/scm/ws_dfu.ecm

```diff
@@ -868,6 +868,9 @@ ESPresponse [exceptions_inline] DFUFileAccessResponse
     
     binary RecordTypeInfoBin;   // optional
     string RecordTypeInfoJson;  // optional
+    
+    int fileAccessPort;
```

Comment:
The ECM structure has changed as thus: 
ESPStruct DFUFileAccessInfo
{
    string MetaInfoBlob;
    string ExpiryTime;
    // {NumParts, FileParts} depend on ReturnFileInfo in request
    int NumParts;        // number of parts in logical file
    ESParray<EspStruct DFUPartLocations> FileLocations;
    ESParray<EspStruct DFUFilePart> FileParts;
    int fileAccessPort;
    bool fileAccessSSL;

    binary RecordTypeInfoBin;   // optional
    string RecordTypeInfoJson;  // optional
};

ESPresponse [exceptions_inline] DFUFileAccessResponse
{
    EspStruct DFUFileAccessInfo AccessInfo;
};
I assume these 2 fields will be part of the DFUFileAccessInfo

Reply:
Depending on which PR merges first.. yes someone should merged them info DFUFileAccessInfo